src/Resources/config/services.xml

```diff
@@ -5,21 +5,24 @@
            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
     <services>
-        <service id="knp_disqus.disqus_client" class="Knp\Bundle\DisqusBundle\Client\DisqusClient">
+        <service id="knp_disqus.config" class="Knp\Bundle\DisqusBundle\Model\DisqusConfig">
             <argument/> <!-- $apiKey -->
             <argument/> <!-- $secretKey -->
             <argument>%kernel.debug%</argument>
         </service>
+        <service id="knp_disqus.disqus_client" class="Knp\Bundle\DisqusBundle\Client\DisqusClient">
+            <argument type="service" id="knp_disqus.config"/>
+        </service>
 
         <service id="Knp\Bundle\DisqusBundle\Client\DisqusClientInterface" alias="knp_disqus.disqus_client" />
 
         <service id="knp_disqus.helper" class="Knp\Bundle\DisqusBundle\Helper\DisqusHelper">
             <argument type="service" id="twig"/>
+            <argument type="service" id="knp_disqus.config"/>
             <argument type="service" id="knp_disqus.disqus_client"/>
-            <argument>%kernel.environment%</argument>
```

Comment:
Btw, this fixes a bug that caused the debug info to be printed in `prod` mode. This should have been `%kernel.debug%`. This caused "debug" to always appear "truthy".

Reply:
Yeah I found it too when examined code